### DISCUSSION TITLE
Bring back saved runes ( dev: use cookies OR localStorage )

### DISCUSSION THREAD

**Discussion by Pr0ph3cyX** [fabd/diablo2-runewizard#14] [3. BUGS & DATA CORRECTIONS]
Awhile back you could save have many runes you have by leaving the website. Now it resets and I have to click each rune every time.

please fix

**Comment by fabd** [comment]
Hya

Hmm.. the runes are saved in [local storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) instead of cookies. Do you have an extension perhaps that clears all "local" or "offline data" when you exit browser?

What device and exact browser are you using?

I mean I could revert to saving in cookies, but we'll need to figure out what pros/cons in each case.
